iac/environments/sbx1/backend.tf

```diff
@@ -0,0 +1,6 @@
+terraform {
+  backend "s3" {
+    region = "eu-west-2"
+    key    = "ccs-scale-agreements-service-sbx1"
```

Comment:
Just wondering - looks like we need to differentiate the keys in the backend files in `ccs-scale-shared-paas-infra` - and do we want to follow the same naming strategy (i.e. `namespace/[infra|service]` ?

Reply:
Ignore the above - I see you have updated all the keys in the shared infra PR - nice one 👍